pkg/kv/kvclient/kvcoord/txn_coord_sender.go

```diff
@@ -1209,6 +1213,7 @@ func (tc *TxnCoordSender) checkTxnStatusLocked(ctx context.Context, opt kv.TxnSt
 		// Nothing to check.
 	case kv.OnlyPending:
 		// Check the coordinator's proto status.
+		// todo fix
```

Comment:
Do we need to fix this?

Reply:
whoops that was my old scratch note when i was using the debugger